api-extractor/src/definitions/ApiItem.ts

```diff
@@ -307,18 +319,12 @@ abstract class ApiItem {
    * This function assumes all references from this ApiItem have been resolved and we can now safely create
    * the documentation.
    */
-  protected onResolveReferences(): void {
-    let originalJsDoc: string = '';
-    if (this.jsdocNode) {
-      originalJsDoc = TypeScriptHelpers.getJsDocComments(this.jsdocNode, this.reportError);
-    }
+  protected onCompleteReferences(): void {
 
-    this.documentation = new ApiDocumentation(
-      originalJsDoc,
-      this.extractor.docItemLoader,
-      this.extractor,
-      this.reportError
-    );
+    // Ensure links are valid
+    this.documentation.completeLinks();
+    // Ensure inheritdocs are valid
+    this.documentation.completeInheritdocs();
```

Comment:
Maybe we should just do `this.documentation.completeReferences()` and then make completeLinks/completeInheritdocs a private implementation detail?